x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/protection_setting_card_switch.tsx

```diff
@@ -82,6 +82,11 @@ export const ProtectionSettingCardSwitch = React.memo(
                 newPayload[os].popup[protection as LinuxPolicyProtection].enabled =
                   event.target.checked;
               }
+              if (protection === 'behavior_protection') {
+                newPayload.windows.behavior_protection.reputation_service = event.target.checked;
```

Comment:
nit: `event.target.checked` is always `false` here. you could simply set the attributes to `false` to simplify reading the code

Reply:
Just went with the the approach used above, sure, it makes no sense, will change that :)